imagebuildah/stage_executor.go

```diff
@@ -412,6 +412,16 @@ func (s *StageExecutor) Copy(excludes []string, copies ...imagebuilder.Copy) err
 						} else {
 							contextDir = additionalBuildContext.DownloadedCache
 						}
+					} else {
+						// This points to a path on the filesystem
+						// Check to see if there's a .containerignore
+						// file, update excludes for this stage before
+						// proceeding
+						buildContextExcludes, _, err := parse.ContainerIgnoreFile(additionalBuildContext.Value, "", nil)
+						if err != nil {
+							return err
+						}
+						excludes = append(excludes, buildContextExcludes...)
```

Comment:
I'm on the fence and looking for other opinions.  Here: https://github.com/containers/buildah/blob/b90c36c7ed30af1cf04c0389e95b0515c8e6bef3/imagebuildah/stage_executor.go#L351 we have a couple of Debug lines to show what's being excluded.  I think it might be good to move those somewhere below this change.

Reply:
Sounds reasonable. Are you referring to this block inside `Copy()`?
```go
		if copy.Download {
			logrus.Debugf("ADD %#v, %#v", excludes, copy)
		} else {
			logrus.Debugf("COPY %#v, %#v", excludes, copy)
		}
```

Reply:
bump! cc/ @rhatdan

Reply:
Moved it to the tail end of the loop just before we invoke the builder with opts.